extensions/Aness6040/Aness_Minecraft_Utils.js

```diff
@@ -0,0 +1,257 @@
+// Name: Minecraft Utils
+// ID: minecraftUtils
+// Description: Adds detections to the Minecraft game for example if the player is a premium account or get the icon of a Minecraft server in URI etc...
+// By: Aness6040 <https://scratch.mit.edu/users/AnessScratched/>
+// Creator's Github: https://github.com/Aness6040
```

Comment:
Listing your GitHub isn't exactly necessary, since the extension is hosted on GitHub anyways. If someone needs your GitHub to contact you, they can find it through the repo.